RELEASING.adoc

```diff
@@ -0,0 +1,12 @@
+= Spring Cloud GCP release process
+
+The actual release process is triggered by Pivotal.
```

Comment:
This is something obvious. The project is owned by Pivotal per se.
I fully don't think that there should be something to tell like "ask Pivotal" in the project files.

I don't mind about such an instruction, but it must be as formal as possible and should come from the project perspective. You can have this instruction on your, Google side, but not here in the Pivotal project.